src/lib/navigation/navigation-mode-selection.spec.ts

```diff
@@ -89,13 +70,4 @@ describe('NavigationMode', () => {
     wizard.updateNavigationMode(navigationMode);
     expect(wizard.navigation).toEqual(navigationMode);
   });
-
-  it('can be assigned with updateNavigationMode by name', () => {
-    const wizardTestFixture = TestBed.createComponent(WizardTestComponent);
-    wizardTestFixture.detectChanges();
-    const wizard = wizardTestFixture.componentInstance.wizard;
-
-    wizard.updateNavigationMode('free');
-    expect(wizard.navigation).toEqual(jasmine.any(FreeNavigationMode));
-  });
 });
```

Comment:
Unit test removed because the tested function (specifying navigation mode by name) is no longer provided.